src/sentry/backup/services/import_export/impl.py

```diff
@@ -88,6 +92,28 @@ def get_existing_import_chunk(
     )
 
 
+def fixup_array_fields[T: (str, str | bytes)](json_data: T) -> T:
+    # preserve for 3 versions as per https://docs.sentry.io/concepts/migration/#version-support-window
+    # so probably 2025.09 this can go away?
```

Comment:
We'll probably need to keep this around much longer, as sales often works with customers on much older sentry versions :cry: